sgkit/io/vcf/vcf_reader.py

```diff
@@ -20,6 +20,7 @@
 import numpy as np
 import xarray as xr
 from cyvcf2 import VCF, Variant
+from numcodecs import Blosc, PackBits
```

Comment:
This can be problematic from experience - I'm not sure about Blosc, but Zstd can fail to import from numcodecs if there's a compile problem. Unfortunately, early on numcodecs allowed installations to proceed even if some bits failed to compile, and they're stuck with that behaviour now :(

https://github.com/tskit-dev/tsinfer/issues/161

Reply:
Ah, I wasn't aware of that. I've put the import for Blosc in a try/except block now, which issues a warning and falls back to no compression if the import fails.